device-types/HPE/FlexNetwork-5130-48G-PoEP-4SFPP-HI.yml

```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5130-48G-POE+-4SFP+-1-slot HI
+slug: hpe-flexnetwork-5130-48G-POEP-4SFPP-HI
+part_number: JH326A
+u_height: 1
+is_full_depth: false
+console-ports:
```

Comment:
Would you be able to add a comment to the datasheet, as well as weight and airflow if you can find that information? Thank you!